test/box-luatest/gh_7231_memtx_hash_idx_gt_iter_deprecation_test.lua

```diff
@@ -0,0 +1,41 @@
+local fio = require('fio')
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new {
+        alias   = 'dflt',
+        box_cfg = {memtx_use_mvcc_engine = true}
+    }
+    cg.server:start()
+    cg.server:exec(function()
+        local s = box.schema.create_space('s')
+        s:create_index('pk', {type = 'HASH'})
+    end)
+end)
+
+g.after_all(function(cg)
+    cg.server:drop()
+end)
+
+--[[
+Checks that a deprecation warning is printed exactly once when using memtx HASH
+index 'GT' iterator type.
+]]
```

Comment:
Nit: We format multi-line comments like this usually:

```Lua
-- Checks that a deprecation warning is printed exactly once when using memtx HASH
-- index 'GT' iterator type.
```

Reply:
Okay, fixed: I will stick to this style in the future.